src/web/app/pages-instructor/instructor-search-page/instructor-search-bar/instructor-search-bar.component.ts

```diff
@@ -9,8 +10,14 @@ import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
   styleUrls: ['./instructor-search-bar.component.scss'],
 })
 export class InstructorSearchBarComponent implements OnInit {
-  @Input() searchKey: string = '';
-  @Output() searched: EventEmitter<string> = new EventEmitter<string>();
+
+  @Input() searchParams: SearchParams = {
+    searchKey: '',
```

Comment:
Actually there is a bad pattern here where the child component changes the value from the parent component.